Source/monster.cpp

```diff
@@ -4486,7 +4485,7 @@ void MissToMonst(Missile &missile, Point position)
 	if (dMonster[oldPosition.x][oldPosition.y] <= 0)
 		return;
 
-	Monster &target = *MonsterAtPosition(oldPosition);
+	Monster &target = *FindMonsterAtPosition(oldPosition);

```

Comment:
Just noticed this can use `FindMonsterAtPosition(oldPosition, true)`

Reply:
True, fixed on 039e7846cb2a5b6d5eb8318b2f035ce690967e56